.github/workflows/pipeline.yml

```
@@ -0,0 +1,383 @@
+        git commit -m "Format black" -a
+    - name: push
+      uses: ad-m/github-push-action@master
+      with:
+        github_token: ${{ secrets.DEPENDABOT_WORKFLOW_TOKEN }}
+        branch: ${{ github.event.pull_request.head.ref }}
+
+  mypy:
+    needs: [black]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Python
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)